README.md

```diff
@@ -48,6 +48,11 @@ etcd_download_dir: "/opt/etcd"
 etcd_bin_dir: "/usr/local/bin"
 # etcd data directory (etcd database files so to say)
 etcd_data_dir: "/var/lib/etcd"
+# Architecture to download and install
+etcd_architecture: "amd64"
+# Only change this if the architecture you are using is unsupported (for example: arm64)
+# For more information, see this: https://github.com/etcd-io/etcd/blob/master/server/etcdmain/etcd.go#L460
```

Comment:
That link is already outdated :wink: I think it makes sense to point to the `op-guide`:
```suggestion
# For more information, see this: https://github.com/etcd-io/website/blob/master/content/docs/v3.4/op-guide/supported-platform.md
```